src/lib/cache/subCache.cpp

```diff
@@ -1285,3 +1306,33 @@ void subCacheStart(void)
   }
   pthread_detach(tid);
 }
+
+
+
+/* ****************************************************************************
+*
+* subCacheItemErrorStatus - 
+*/
+void subCacheItemErrorStatus(const char* tenant, const char* subscriptionId, int errors)
```

Comment:
When the sub-cache was rewritten from scratch (due to inexplicable error that later turned out to be mongo exceptions), it was rewritten entirely in C to avoid strange stuff.
So, most of the sub cache (I thought all of it) uses char* and not std::string.
However, I checked and it turns out std::string is used in around 50% of the code of the sub-cache anyway.

Changed to std::string in 75b1df3